core/chain-configs/src/genesis_config.rs

```diff
@@ -184,13 +188,26 @@ pub struct GenesisConfig {
     pub minimum_validators_per_shard: NumSeats,
     #[cfg(feature = "protocol_feature_max_kickout_stake")]
     #[serde(default = "default_max_kickout_stake_threshold")]
-    #[default(30)]
+    #[default(100)]
+    /// Max stake percentage of the validators we will kick out.
     pub max_kickout_stake_perc: u8,
     /// The lowest ratio s/s_total any block producer can have.
     /// See https://github.com/near/NEPs/pull/167 for details
     #[serde(default = "default_minimum_stake_ratio")]
     #[default(Rational32::new(160, 1_000_000))]
     pub minimum_stake_ratio: Rational32,
+    #[serde(default = "default_use_production_config")]
+    #[default(false)]
+    /// This is only for test purposes. We hard code some configs for mainnet and testnet
+    /// in AllEpochConfig, and we want to have a way to test that code path. This flag is for that.
+    /// If set to true, the node will use the same config override path as mainnet and testnet.
+    pub use_production_config: bool,
```

Comment:
btw - maybe this shouldn't be public ? (as we want people to use the fun use_production_config instead).

Reply:
Yeah I would like that too, but unfortunately some traits that this struct derives requires all field to be public